frontend/apps/web/components/FileUpload/FileUpload.tsx

```diff
@@ -231,8 +235,9 @@ function UploadedFiles<T>(props: UploadedFilesProps<T>): ReactElement {
         <div className="mt-4 space-y-2">
           {files.map((file, index) => (
             <div key={index} className="flex flex-col space-y-2">
-              <div className="flex items-center justify-between p-3 bg-white rounded-lg border">
+              <div className="flex items-center justify-between p-3 bg-white dark:bg-gray-700 rounded-lg border dark:border-0">
```

Comment:
curious how this looked if we just removed the `bg-white` and used the default. Either way, this looks great!

Reply:
shouldn't make a difference since the default is white anyways